src/ShearStressTransportEquationSystem.C

```diff
@@ -554,5 +578,64 @@ ShearStressTransportEquationSystem::compute_f_one_blending()
   }
 }
 
+//------------------------------------------------------------------------------
+//--------- compute_f_ABLBlending ----------------------------------------------
+//------------------------------------------------------------------------------
+void 
+ShearStressTransportEquationSystem::compute_f_ABLBlending()
+{
+
+  stk::mesh::MetaData &meta_data = realm_.meta_data();
+ 
+  const double power    = realm_.get_turb_model_constant(TM_power);
+  const double bnd_dist = realm_.get_turb_model_constant(TM_bnd_dist);
+  const double sp_dist  = realm_.get_turb_model_constant(TM_sp_dist);
+
+  stk::mesh::Selector s_all_nodes
+    = (meta_data.locally_owned_part() | meta_data.globally_shared_part())
+    &stk::mesh::selectField(*fABLBlending_);
+
+  stk::mesh::BucketVector const& node_buckets =
+    realm_.getbuckets( stk::topology::NODE_RANK, s_all_nodes );
+
+  for ( stk::mesh::BucketVector::const_iterator ib = node_buckets.begin() ;
+        ib != node_buckets.end() ; ++ib) {
+    stk::mesh::Bucket & b = **ib ;
+    const stk::mesh::Bucket::size_type length  = b.size();
+
+
+   const double * minD = stk::mesh::field_data(*minDistanceToWall_,b);  
+   const double * tmpf = stk::mesh::field_data(*fABLBlending_, b);
+   
+   for ( stk::mesh::Bucket::size_type k = 0 ; k < length ; ++k) {
+
+
+      const double signum = 0.0;
```

Comment:
you are trying to change `signum` below, so you won't want to define it as a `const`